build/stf-run-ci/tasks/setup_stf.yml

```diff
@@ -9,7 +9,7 @@
         namespace: openshift-marketplace
       spec:
         displayName: InfraWatch Operators
-        image: quay.io/infrawatch-operators/infrawatch-catalog:unstable
+        image: quay.io/infrawatch-operators/infrawatch-catalog:nightly
```

Comment:
Unstable is not a thing, so I fixed this to match a tag that gets updated regularly. Not sure how often this path is used, but it would not have worked previously.

Reply:
I had no idea that this was here. 

Is this related to index deployment?

If so, it doesn't currently run in CI but the CI job is a WIP at #495

Reply:
I'm just cleaning it up because I was here, but I think it's a legacy interface and I wouldn't necessarily expect CI to test for it.